clang-16 refuses to compile code that uses certain extensions that were before
implicitly enabled.
g++ also rejects it.
add the required extensions to compile the vectorised code with runtime detection.
the code already does this, but these -Xclang args don't work for some reason
--
diff --git a/src/3rdparty/chromium/third_party/crc32c/BUILD.gn b/src/3rdparty/chromium/third_party/crc32c/BUILD.gn
index 39c151a5b..702e8b29f 100644
--- a/src/3rdparty/chromium/third_party/crc32c/BUILD.gn
+++ b/src/3rdparty/chromium/third_party/crc32c/BUILD.gn
@@ -109,23 +109,7 @@ source_set("crc32c_arm64") {
   if (current_cpu == "arm64") {
     if (is_clang) {
       cflags = [
-        "-march=armv8-a",
-
-        # Some builds set -march to a different value from the above.
-        # The specific feature flags below enable the instructions we need
-        # in these cases. See https://crbug.com/934016 for example.
-        "-Xclang",
-        "-target-feature",
-        "-Xclang",
-        "+crc",
-        "-Xclang",
-        "-target-feature",
-        "-Xclang",
-        "+crypto",
-        "-Xclang",
-        "-target-feature",
-        "-Xclang",
-        "+aes",
+        "-march=armv8-a+crc+crypto",
       ]
     } else {
       cflags = [ "-march=armv8-a+crc+crypto" ]
